ws-nextjs-app/pages/[service]/live/[id]/Header/index.tsx

```diff
@@ -83,20 +87,24 @@ const Header = ({
               css={[
                 styles.description,
                 showLiveLabel &&
-                  !isHeaderImage &&
+                  !isWithImageLayout &&
                   styles.layoutWithLiveLabelNoImage,
               ]}
             >
               {description}
             </Text>
           )}
-          {mediaCollections && (
+        </div>
+        {mediaCollections && (
+          <div
+            css={[styles.liveMediaClose, isMediaOpen && styles.liveMediaOpen]}
```

Comment:
This is a little bit confusing to read - and you could end up with `liveMediaClose` and `liveMediaOpen` as the styles in a particular scenario. Would it still look right if we were to use a ternary expression? `isMediaOpen ? styles.liveMediaOpen : styles.liveMediaClose`? 

Otherwise, should we consider renaming the `liveMediaClose` css function if it's always going to be used?